src/Utilities/UnitTests/ToolLocationHelper_Tests.cs

```diff
@@ -103,6 +103,55 @@ public void GetApiContractReferencesFindsVersionedWinMDs()
         }
 
         [Fact]
+        public void GetWinBlueSDKLocation()
+        {
+            string sdkRootPath = ToolLocationHelper.GetPlatformSDKLocation("Windows", "8.1");
+
+            string returnValue = ToolLocationHelper.GetSDKContentFolderPath("Windows", "8.1", null, null, null, null);
+            Assert.Equal(returnValue, sdkRootPath);
```

Comment:
The Assert at this line still looks backwards.  The returnValue is _actual_ right?